src/resolvers/tag.ts

```diff
@@ -40,11 +40,24 @@ export async function tagsSavedItems(
     }
   }
   // Now get result
-  const res = await savedItemDataService.getSavedItems(
-    args.filter,
-    args.sort,
-    args.pagination,
-    parent.savedItems
-  );
-  return res;
+  // If the IDs are on the parent, use them
+  if (parent.savedItems != null) {
+    return savedItemDataService.getSavedItems(
+      args.filter,
+      args.sort,
+      args.pagination,
+      parent.savedItems
+    );
+  } else {
+    // Use filter to retrieve the SavedItems if IDs are not on the parent
```

Comment:
suggested tags won't get the list of ids, but we can easily retrieve the result we need with a filter